src/Sulu/Bundle/CustomUrlBundle/Resources/js/containers/Form/fields/CustomUrlsLocaleSelect.js

```diff
@@ -26,19 +15,15 @@ class CustomUrlsLocaleSelect extends React.Component<FieldTypeProps<string>> {
     };
 
     render() {
-        const {disabled, value} = this.props;
-
-        if (!this.webspace) {
-            return null;
-        }
+        const {disabled, formInspector, value} = this.props;
 
         return (
             <SingleSelect
                 disabled={!!disabled}
                 onChange={this.handleChange}
                 value={value}
             >
-                {this.webspace.allLocalizations.map(({localization}) => (
+                {webspaceStore.getWebspace(formInspector.options.webspace).allLocalizations.map(({localization}) => (
```

Comment:
same here `webspaceStore.getWebspace(formInspector.options.webspace).allLocalizations` would this not make sense to be a computed value?

Reply:
same as above would move to `@computed` if render to avoid this on every render call